packages/@aws-cdk/aws-codestarnotifications/lib/notify-on-event-options.ts

```diff
@@ -0,0 +1,50 @@
+import { Status, DetailType } from './rule';
+import { IRuleTarget } from './target';
+
+/**
+ * Standard set of options for `notifyOnXxx` codestar notification handler on construct
+ */
+export interface NotifyOnEventOptions {
```

Comment:
I would move this interface to `rule.ts`, I don't think it deserves its own file.

Reply:
Already moved to `rule.ts`.